fendermint/app/src/settings.rs

```diff
@@ -68,6 +71,10 @@ pub struct Settings {
     pub db: DbSettings,
     pub eth: EthSettings,
     pub fvm: FvmSettings,
+
+    #[serde(deserialize_with = "deserialize_subnet_id", default)]
```

Comment:
What does the `default` do? If there is a default here, why do we need one in the `default.toml` file?

Reply:
This was copy pasted, should have removed the `default`